<ui version="4.0" >
 <class>PresentationCtrlWidget</class>
 <widget class="QWidget" name="PresentationCtrlWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>77</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <widget class="QLabel" name="m_slideLabel" >
     <property name="text" >
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QPushButton" name="m_playButton" >
       <property name="text" >
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="m_stopButton" >
       <property name="text" >
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QPushButton" name="m_prevButton" >
       <property name="text" >
        <string>Prev</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QPushButton" name="m_nextButton" >
       <property name="text" >
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
